v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/constants.py

```diff
@@ -166,6 +166,9 @@
 FM_ACCOUNT_USERNAME: str = ""
 FM_ACCOUNT_PASSWORD: str = ""
 
+# Name of the FM asset that is managed, e.g. "John's Quasar"
+FM_ASSET_NAME: str = ""
```

Comment:
How will this get the initial value?
What is the added value of adding this to V2G Liberty?

Reply:
The asset name is needed for retrieving the asset_id. That is needed for sending the asset attributes to FM.